SU2_CFD/include/solvers/CSolver.hpp

```diff
@@ -4469,4 +4469,16 @@ class CSolver {
     }
   }
 
+  /*! 
+   * \brief LUT premixed flamelet: Set the number of table look up misses.
+   * \param[in] val_n_table_miss - Number of table look up misses.
+   */
+  inline virtual void SetNTableMisses(unsigned short val_n_table_miss) { }
+
+  /*! 
+   * \brief LUT premixed flamelet: Get the number of table look up misses.
+   * \return Number of table look up misses.
+   */
+  inline virtual unsigned long GetNTableMisses() { return 0; }
+
```

Comment:
This looks unused, at least setting the number from the outside world should not make sense I think.
```suggestion
```